openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -0,0 +1,74 @@
+from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import (
+    CreatedInstance,
+    OpenPypePyblishPluginMixin
+)
+
+
+class CreateReview(plugin.HoudiniCreator, OpenPypePyblishPluginMixin):
+    """Review with OpenGL ROP"""
+
+    identifier = "io.openpype.creators.houdini.review"
+    label = "Review"
+    family = "review"
+    icon = "video-camera"
+
+    # Default settings for the ROP
+    # todo: expose in OpenPype settings?
+    override_resolution = True
+    width = 1280
+    height = 720
+    aspect = 1.0
+
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        import hou
```

Comment:
why not a global import?

Reply:
This actually happens across the code base that some imports happen in the pyblish plugins. I am not entirely sure why we would for `hou` so yes we can make this global.

I do wonder also why it's done in most others too. I vaguely recall that in very early versions of Pyblish there was a bug that some of the imported libs would already get garbage collected in the plugin files before process ran or something, but that was a long time ago that I had seen anything like that.

Reply:
with those host specific modules the reason is mostly that it make discovery of the plugins slow so we move it usually to the place where it is directly used.

Reply:
awesome, good to know, thank you!